src/components/Meta.js

```diff
@@ -59,24 +58,6 @@ export default class Meta extends Component {
         <meta property="og:image:secure_url" content={absoluteImageUrl} />
         <meta property="og:image" content={absoluteImageUrl} />
         <meta name="twitter:card" content={absoluteImageUrl} />
```

Comment:
Are these relevant to the project going forward. When is twitter card used? or is this leftover from when the template's social media links.

Reply:
I think it's just a remnant. I wasn't sure if we were keeping any future social media integration in mind so I just kept it there, but we could just take it out for now.

Reply:
I see. I recall the library does have a Youtube channel, so keeping these as reference when adding that in wouldn't be a bad idea.